client/v2/internal/testpb/query.proto

```diff
@@ -10,10 +10,10 @@ import "cosmos/base/v1beta1/coin.proto";
 
 service Query {
   // Echo returns the request in the response
-  rpc Echo(EchoRequest) returns (EchoResponse);
+  rpc Echo(QueryEchoRequest) returns (QueryEchoResponse);
 }
 
-message EchoRequest {
+message QueryEchoRequest {
```

Comment:
I don't think this is required. Running `make proto-all` does not tell me to change anything in this file.
Could you revert?

Reply:
yes, sure, will fix right away

Reply:
fixed everything @julienrbrt